src/Neo.VM/JumpTable/JumpTable.Stack.cs

```diff
@@ -64,7 +64,7 @@ public virtual void Nip(ExecutionEngine engine, Instruction instruction)
         /// </summary>
         /// <param name="engine">The execution engine.</param>
         /// <param name="instruction">The instruction being executed.</param>
-        /// <remarks>Pop 1, Push 0</remarks>
+        /// <remarks>Pop 2, Push 0</remarks>
```

Comment:
It's misleading to me. "Pop" usually refers to the top of the stack, in that sense XDROP pops _one_ element to drop another one. Take NIP for example, it drops something below as well, but it doesn't pop anything. If we want to express changes to the depth of the stack that'd be minus two elements, true.